### PR TITLE
insert current gui selection at point

### PR DESCRIPTION
**Pull request by jll63**

### CODE REVIEW

**Comment by jll63** [comment]
assign: @philippe-grenet

File: modules/init-look-and-feel.el [hunk 1]
```diff
@@ -167,6 +167,17 @@
   (when (boundp 'global-undo-tree-mode)
     (global-undo-tree-mode -1)))
 
+(defun insert-gui-primary-selection ()
+  "If no region is selected, insert current gui selection at point."
+  (interactive)
+  (when (not (use-region-p))
+    (let ((text (gui-get-selection)))
+      (push-mark (point))
+      (insert-for-yank text))))
```
Comment (pkryger) [inline modules/init-look-and-feel.el]: I think `insert-for-yank` requires the arg to be not `nil`. How about checking it before executing the `let` body?

Comment (jll63) [reply modules/init-look-and-feel.el]: Done.

**Comment by philippe-grenet** [comment]
Sorry I forgot this PR.

**Pull request merged**